docs/contributing/database-migrations/ef.md

```diff
@@ -4,11 +4,20 @@ sidebar_position: 2
 
 # Entity Framework
 
+:::info
+
+For instructions on how to apply database migrations, please refer to the
+[Getting Started](../../getting-started/server/database/ef/index.mdx) documentation.
+
+:::
+
 If you alter the database schema, you must create an EF migration script to ensure that EF databases
 keep pace with these changes. Developers must do this and include the migrations with their PR.
 
 To create these scripts, you must first update your data model in `Core/Entities` as desired. This
-will be used to generate the migrations for each of our EF targets.
+will be used to generate the migrations for each of our EF targets. Additionally, for table changes
+it is strongly recommended to define or update an `IEntityTypeConfiguration<T>` to accurately
```

Comment:
This seems out of place, it's not directly related to migrations but general EF best practices.

Reply:
Where would it go then? I feel like the prior existing sentence is in the same camp, but both need to be modified for migrations so I put it here.